spec/pluginmanager/commands/install_spec.rb

```diff
@@ -0,0 +1,68 @@
+# encoding: utf-8
+require "spec_helper"
+require "pluginmanager/commands/install_command"
+require "uri"
+require 'webmock/rspec'
+
+describe LogStash::PluginManager::InstallCommand do
```

Comment:
version here does not mather, is just to write the test and validate them, nothing else. I would rather not update it or interpolate any LOGSTASH_VERSION, but this is a matter of taste to me.